modules/node_modules/@ncigdc/components/Header.js

```diff
@@ -28,7 +28,7 @@ const styles = {
 const Header = compose(
   connect(state => ({
     notifications: state.bannerNotification,
-    loggedIn: state.auth.isAuthorized,
+    loggedIn: state.auth.user && Object.keys(state.auth.user.projects.gdc_ids || {}).length > 0,
```

Comment:
oh you don't need that projects check here. on the portal you're either logged in or you're not. submission-ui had more specifics than that